css/_side_toolbar_container.scss

```diff
@@ -2,7 +2,7 @@
  * Toolbar side panel main container element.
  */
 #sideToolbarContainer {
-    background-color: rgba(0,0,0,0.8);
+    background-color: $sideBarBackground;
```

Comment:
When moving the side toolbar to react it was renamed to SecondaryToolbar, but the styles weren't changed, so now there's inconsistency in the names. I can change my var to secToolbarContainerBackground, but I'm not sure if this is an improvement. I don't want to fix styling names of existing code right now.